data/maps/LittlerootTown/scripts.inc

```diff
@@ -48,18 +48,22 @@ LittlerootTown_EventScript_1E7E5C:: @ 81E7E5C
 	setvar VAR_0x408C, 5
 	return
 
-LittlerootTown_EventScript_1E7E67:: @ 81E7E67
+@ Mom's default coords (in map.json) are at brendan's house.
+@ This will permamently move her to may's house.
+LittlerootTown_EventScript_MoveMomToMaysHouse:: @ 81E7E67
 	setobjectxyperm 4, 14, 8
 	return
 
-LittlerootTown_EventScript_1E7E6F:: @ 81E7E6F
-	compare VAR_0x4050, 0
-	goto_if_eq LittlerootTown_EventScript_1E7E86
+LittlerootTown_EventScript_SetTwinPosition:: @ 81E7E6F
+	compare VAR_BIRCH_ATTACK, BIRCH_ATTACK_INACTIVE
```

Comment:
In pokeruby, your `VAR_BIRCH_ATTACK` var is called `VAR_LITTLEROOT_STATE`.  I'm worried that other vars in this PR deviate significantly from pokeruby.  In my opinion, `VAR_LITTLEROOT_STATE` is better for this var, since this is not specific to the Birch attack.